cmd/kyma/alpha/list/module/module.go

```diff
@@ -141,6 +148,15 @@ func (cmd *command) run(ctx context.Context, kymaName string) error {
 		}
 	}
 
+	if cmd.RESTMapper == nil {
```

Comment:
Isn't `cmd.RESTMapper` always `nil` here?

Reply:
Normally yes. But in the tests, we have to inject a RESTMapper here. Otherwise I would have to mock a KubeConfig and for that I would need a live cluster for the discovery to work. That's why I decided to introduce this if statement. In the tests this logic will be skipped if a RESTMapper is provided.